portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,22 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let index = 0;
+function showNext() {
+    index = index +1;
+    if(index ==2){
```

Comment:
Space after `+` and `==`, and before `{`.

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace
#3: "Before any open curly brace"
#4: "On both sides of any binary or ternary operator"